docs/authentication.md

```diff
@@ -1,59 +1,83 @@
 
-### Login
-There are three ways users can login to Artsy.
-  1. Email
-  2. Facebook
-  3. Twitter (signups deprecated)
+There are four types of forms related to authentication: `singup`, `login`, `forgot` and `reset_password`. Analytics are built into all forms for `authImpression` events. [See the full list of options passed to Forms here.](https://github.com/artsy/reaction/blob/master/src/Components/Authentication/Types.ts#L45)
 
-### Signup
-There are two ways users can signup to Artsy. Twitter signups have been deprecated for a while and we intend to convert existing Twitter users to email users eventually.
-  1. Email
-  2. Facebook
+The only form not located in Reaction is `reset_password`, which currently exists as a coffeescript template in Force.
 
-New signups require a name, email, 8 character password minimum, and they must accept our terms, policies and to receive emails.
+There are two ways users can create an Artsy account, with an email address or via their Facebook account. Previously we supported auth via Twitter and LinkedIn, these services are now deprecated.
+
+New signups require:
+
+- Name
+- Email
+- Password (8 character minimum)
+- Accepted terms and conditions, and to receive emails.
 
 ### Forgot Password
+
 If a user forgets their password, they are able to reset it using the Forgot Password form. This form only accepts an email address. Reset instructions will be sent to the user along with a reset token which is used to verify the user.
 
 Note: In Reaction, we incorrectly use the name ResetPassword.
 
 ### Reset Password
+
 This is the only form that exists as a coffeescript template in Force. To successfully reset your password, the form must be submitted with a valid token (typically taken from a query param). Below are the allowed query params for /reset_password:
 
 - reset_password_token: the token which authorizes you to change your password
 - reset_password_redirect_to: where the user should go after resetting their password
 - set_password: flag to change copy to distinguish between setting a new password entirely and changing the existing password
 
 ## Entry Points
+
 There are three ways a user can see authentication forms. Note that each of these are using the same underlying form infrastructure which includes validation, form submission, and analytics events.
 
 ### Modal
-Authentication modals can only be triggered with the `mediator`. The `mediator` is a global event emitter/listener in Force that is responsible for various event handling. Below is an example of how a modal might be triggered.
+
+There are helpers located in Reaction and Force to open authentication modals. Use one of Reaction's helpers if a `mediator` is available to your component.
+
+It is recommended that the helper is used rather than calling the `mediator` directly to ensure that all typing for analytics is available.
```

Comment:
This is a good long-term change towards factoring out `mediator`, too, since  `mediator` depends on Backbone.

Reply:
Yes, one of the goals here is to standardize how we're calling `mediator`, which will make it easier to deprecate -- this is the long goal.